renderdoc/driver/ihv/amd/amd_counters.h

```diff
@@ -25,20 +25,29 @@
 #pragma once
 
 #include <vector>
+#include <map>
 #include "api/replay/renderdoc_replay.h"
+#include "api/replay/data_types.h"
 
 struct _GPAApi;
 
 inline constexpr GPUCounter MakeAMDCounter(int index)
 {
   return GPUCounter((int)GPUCounter::FirstAMD + index);
 }
-
 class AMDCounters
 {
 public:
+  enum ApiType
+  {
+    eApiType_Dx11 = 0,
+    eApiType_Dx12 = 1,
+    eApiType_Ogl = 2,
+    eApiType_Vk = 3
+  };
```

Comment:
It's not a hard rule but I tend to prefer `enum class` now that the code is compiled as C++11, so e.g. `enum class AMDCounterType { D3D11, D3D12, GL, Vulkan }` at global scope, or else you could re-use e.g. `enum class GraphicsAPI` which already has these values defined for you, since this doesn't have to match up to an internal enum.

Reply:
Fixed